development/getting-started.md

```diff
@@ -16,136 +17,7 @@ The application suite consists of three core components:
 
 `Client` - The Google Earth Enterprise Client (EC) and Google Maps Javascript API V3 used to view 3D globes and 2D maps, respectively.
 
```

Comment:
Maybe re-phrase:
The application suite consists of four core components:

`Fusion` - Builds 3D Earth and 2D Map from imagery, vector and terrain source data. 
 
 `Earth Enterprise Server` - Apache -based server which hosts 3D Earth, 2D Map build by Fusion and Portable 3D/2D globes.
 
 `Portable server` - Tornado-based server which hosts  Portable 3D/2D globes in a local-server configuration.

`Client` - The Google Earth Enterprise Client (EC) and Google Maps Javascript API V3 used to view 3D globes and 2D maps, respectively.